doc/selfhosting.md

```diff
@@ -155,9 +163,8 @@ use.
 
 ### S3\_BUCKET
 
-This is the name of the S3 bucket that will be used to store Riju
-build artifacts (aside from Docker images). It needs to be globally
-unique, so `yourname-riju` is a good choice.
+This is the name of the S3 bucket that you created in a previous step.
+`riju-yourname-tf`
```

Comment:
This is actually not true; I've added a new comment about the difference between the two S3 buckets required by Riju in the above-linked commit.